marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/hub-entities/generateSearchOptions.sjs

```diff
@@ -409,6 +409,39 @@ function verifySnippetOptions() {
   ]
 }
 
+function verifySourceNameAndSourceTypeOptions() {
```

Comment:
I don't think this is the right way to test this, because we're not programmatically generating any options - we're just hardcoding two new options in.

Instead, I think we need an ml-test that loads a couple documents - one with sourceName/sourceType, the other without - and then use search:search with the installed options. Then verify that the search:response has the facets for sourceName/sourceType. That verifies both that the options were generated correctly and that they work.